de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/metadata/MetadataViewDefaultAppearance.java

```diff
@@ -254,4 +255,23 @@ public ImageResource saveToFileIcon() {
     public String saveMetadataToFile() {
         return toolbarMessages.saveMetadataMenuItem();
     }
+    
+    public String loading() {
+        return iplantDisplayStrings.loadingMask();
+    }
+
+    @Override
+    public String backgroudStyle() {
```

Comment:
Not sure if this is an issue (probably not if everything uses the same name), but should this be `backgroundStyle`?